packages/markdown/src/markdown_renderers.ts

```diff
@@ -51,8 +51,14 @@ export function customHeadingRenderer(text: string, level: 1 | 2 | 3 | 4 | 5 | 6
 }
 
 export function parseRepoName(gitRepoUrl: string): string {
-    const parsedRepoUrl = new URL(gitRepoUrl);
-    const path = parsedRepoUrl.pathname.substr(1);
+    // Handling for SSH URLs
```

Comment:
let's add some tests for this so we don't break it again (`./test/markdown_renderers.test.ts`)